src/js/Content/Features/Store/Wishlist/FWishlistStats.js

```diff
@@ -12,7 +12,36 @@ export default class FWishlistStats extends Feature {
 
         this._appInfo = await Page.runInPageContext(() => window.SteamFacade.global("g_rgAppInfo"), null, true);
 
-        HTML.beforeBegin("#wishlist_ctn", '<div id="esi-wishlist-chart-content"></div>');
+        HTML.beforeBegin("#tab_filters",
+            `<div class="filter_tab" id="esi-wishlist-stats">
+                ${Localization.str.wl.label}
+                <img src="https://store.akamai.steamstatic.com/public/images/v6/btn_arrow_down_padded_white.png">
+            </div>`);
+
+        HTML.beforeBegin("#section_filters",
+            `<div class="filter_section" id="esi-wishlist-stats-content">
+                <div class="esi-stat"><span id="esi-stat-price"></span>${Localization.str.wl.total_price}</div>
+                <div class="esi-stat"><span id="esi-stat-count"></span>${Localization.str.wl.in_wishlist}</div>
+                <div class="esi-stat"><span id="esi-stat-onsale"></span>${Localization.str.wl.on_sale}</div>
+                <div class="esi-stat"><span id="esi-stat-noprice"></span>${Localization.str.wl.no_price}</div>
+            </div>`);
+
+        const statsBtn = document.getElementById("esi-wishlist-stats");
+        const statsContent = document.getElementById("esi-wishlist-stats-content");
+
+        statsBtn.addEventListener("click", e => {
+            statsBtn.classList.toggle("hover");
+            statsContent.classList.toggle("hover");
+            e.preventDefault();
+        });
+
+        // capture this event so it doesn't get default prevented
+        document.body.addEventListener("click", ({target}) => {
```

Comment:
If I remove the last parameter it still works. What should happen here?

Reply:
This fixes weird behaviour when you click the stats button, then click the options button, then the stats button again. Essentially this hides the stats popup when clicking the options button, at least this is how I think it should work...